docs/pages/database-access/getting-started.mdx

```diff
@@ -96,10 +98,6 @@ see the AWS [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGui
 
 ## Step 2/3. Set up Teleport
```

Comment:
Could/should this and similar headers like "Install and configure Teleport" be updated to something like "Install the Teleport Database Service"?

Reply:
For this one, I think "Set up Teleport" works because the H2 includes steps for configuring Teleport RBAC as well as the Database Service. I'll look into the others.